payments-model/src/main/java/com/stripe/android/model/CvcCheck.kt

```diff
@@ -9,12 +9,16 @@ enum class CvcCheck(
     Pass("PASS"),
     Fail("FAIL"),
     Unavailable("UNAVAILABLE"),
-    Unchecked("UNCHECKED");
+    Unchecked("UNCHECKED"),
+    Unknown("UNKNOWN");
+
+    val requiresRecollection: Boolean
+        get() = this in setOf(Fail, Unavailable, Unchecked)
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     companion object {
         fun fromCode(code: String?): CvcCheck {
-            return values().firstOrNull { it.code.equals(code, ignoreCase = true) } ?: Unavailable
+            return values().firstOrNull { it.code.equals(code, ignoreCase = true) } ?: Unknown
         }
```

Comment:
Previously, I was defaulting to `Unavailable`. However, iOS treats that case as requiring recollection (see [here](https://github.com/stripe-ios/stripe-ios/blob/cb58f1ee8b37c547df93ce3a6dc58537fd393840/Stripe/PaymentDetails.swift#L225)), so adding a separate `Unknown` case seemed like a good idea.